Dockerfile

```diff
@@ -23,6 +25,11 @@ RUN wget -q https://repo.anaconda.com/miniconda/Miniconda3-4.7.12.1-Linux-x86_64
     echo ". /opt/conda/etc/profile.d/conda.sh" >> ~/.bashrc && \
     echo "conda activate base" >> ~/.bashrc
 
+RUN git clone https://github.com/ENCODE-DCC/kentUtils_bin_v377.git  && \
+    chmod +x kentUtils_bin_v377/bin/bedToBigBed && \
+    mv kentUtils_bin_v377/bin/bedToBigBed /usr/local/bin/ && \
+    rm -rf kentUtils_bin_v377
+
```

Comment:
If I remember correctly using the binary from `bin/bedToBigBed` was what ended up working yeah? If so I don't think we need these lines anymore.